subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperLoggingIntegrationTest.groovy

```diff
@@ -17,7 +17,7 @@
 package org.gradle.integtests
 
 class WrapperLoggingIntegrationTest extends AbstractWrapperIntegrationSpec {
-    def "wrapper does not output anything when executed in quiet mode"() {
+    def "wrapper only renders welcome message when executed in quiet mode"() {
```

Comment:
I wonder how many samples tests this is going to break...

Reply:
So far it didn't break sample tests when running the whole pipeline. I guess some other tests are running before, create the marker file and then there's no more need to render the message.

Reply:
Hm.. cool. I thought adding extra output would surely break our samples.